routes/index.js

```diff
@@ -15,8 +16,92 @@ var mlb_key = 'wxf8qgjxs7ka6ay8ec249etg';
 var nba_key = 'hdgj9e9vs9hquzc6ds22wtdy',
   teams = [];
 
+var nba_team_ids = ["583ec825-fb46-11e1-82cb-f4ce4684ea4c","583ec8d4-fb46-11e1-82cb-f4ce4684ea4c","583ecefd-fb46-11e1-82cb-f4ce4684ea4c"];
 
- // step one. we look in mongo
+
+// PLAYERS ============================================================= //
+
+ // Just pull from API each time for now
+db_players.open(function(err,db_players){
+    db_players.collection('players',function(err,collection){
+      collection.find().toArray(function(err, players) {
+        fetchPlayersFromApi();
+      })
+    }) 
+  }); 
+
+// Why is this only pulling for one team? It should pull three times given the array above.
+  var fetchPlayersFromApi = function(){
+    for (i =0; i < nba_team_ids.length; i++){
+      teamz = nba_team_ids[i];
+
+      request('https://api.sportsdatallc.org/nba-t3/seasontd/2014/reg/teams/'+teamz+'/statistics.json?api_key='+nba_key, function (error, response, body) {
+        if (!error && response.statusCode == 200) {
+          var json_response = (JSON.parse(response.body));
+              players = formatPlayers(json_response);
+          //renderPlayers(players);
+          mongoInsertPlayers(players);
```

Comment:
we should do the insert in a nice format that will be easy for the client front end to parse rather than inserting all this stuff. I was thinking the format would be something like ```id : 123, team: Pelicans, market: Nawlins, players: {object} ```